packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -24,19 +55,18 @@ export async function forceCreateManifest(
     uris.push(sourceUri);
   }
   const sourcePaths = uris.map(uri => uri.fsPath);
+  const inputOptions = {
+    placeHolder: nls.localize('manifest_input_save_placeholder'),
+    prompt: nls.localize('manifest_input_save_prompt')
```

Comment:
nit: constant

Reply:
I'm not opposed to this getting changed, but I think we very rarely use constants when calling nls.localize. Is this something we should go back and clean up everywhere?

Reply:
100%.  There should be a single source of truth for what this string value.  I'm always in favor of letting the computer do the work instead of the dev when it comes to remembering what a string value should be.  Benefits us when referencing the value in code as well as verifying it in test.

Reply:
FWIW I'm also a big fan of doing it different now even if we can't go update all the things.  So even if we haven't done it that way in the past if it's a better way then we should start doing it now and adjust as we go. Also I can get out voted :P

Reply:
@gbockus-sf @randi274 - I made this change for now. Let's try and enforce this for future changes and we should come back and make one sweeping change at a later point. It is really hard to find the telemetry strings every time to query something on appinsights.

Reply:
Cool! Sounds like a worthy effort to me, just wanted to make sure we're ready to enforce it going forward 😎